.pre-commit-config.yaml

```diff
@@ -146,6 +146,11 @@ repos:
   #   rev: 2b2f0c74acdb3de316e23ceb7dd0d7945c354050
   #   hooks:
   #     - id: reorder-python-imports
+  - repo: https://github.com/mwouts/jupytext
+    rev: 79351dfac9fde7c72cc73d553f662b4887bc63f5  # CURRENT_TAG/COMMIT_HASH
+    hooks:
+    - id: jupytext
+      args: [--from, py, --to, md]
```

Comment:
What's the purpose of this?  I don't want duplicates of py -> md in the repo. If it is simply about checking whether it is possible, the runner can take care of this already, no?